system/security/cryptohelper/pke.cpp

```diff
@@ -103,11 +116,22 @@ class CLoadedPublicKeyFromMemory : public CLoadedKey
 class CLoadedPrivateKeyFromFile : public CLoadedKey
 {
 public:
+    CLoadedPrivateKeyFromFile(const char *keyFile, size32_t passPhraseLen, const void *passPhrase)
+    {
+        if (!loadKeyFromFile(keyFile))
+            throwEVPException(0, "CLoadedPrivateKeyFromFile: failed to open private key");
+        MemoryBuffer passPhraseMB(passPhraseLen, passPhrase);
+        RSA *rsaKey = PEM_read_bio_RSAPrivateKey(keyBio, nullptr, passphraseCB, (void *)&passPhraseMB);//Binary passphrase
+        memset(passPhraseMB.bufferBase(), 0, passPhraseLen);//ensure local passphrase cleared from memory
```

Comment:
not convinced this is worth it, since the passphrase will still be in memory in 'passPhrase'.

In fact, there's no need to copy it..
You need to pass in a single ptr, that captures memory + len, hence MemoryBuffer,
but you could do:
```
MemoryBuffer passPhraseMB;
passPhraseMB.setBuffer(passPhraseLen, passPhrase);
```
that will avoid the memcpy.